src/client/cmd/common_cli.cpp

```diff
@@ -102,3 +102,23 @@ mp::ReturnCode cmd::standard_failure_handler_for(const std::string& command, std
 
     return return_code_for(status.error_code());
 }
+
+bool cmd::update_available(const multipass::UpdateInfo& update_info)
+{
+    return update_info.version() != "";
+}
+
+std::string cmd::update_notice(const multipass::UpdateInfo& update_info)
+{
+    if (update_available(update_info))
+    {
+        return "\n** A new Multipass version " + update_info.version() +
+               " is available! **\n"
+               "** Find out more: " +
+               update_info.url() + "\n";
```

Comment:
Not sure of the asterisk treatment here… We don't have it anywhere else on our CLI.

Reply:
Without the message easily drowned in the noise. Design input welcome

Reply:
Maybe wrap the whole thing in `###`?
```
###
A new Multipass version…
###
```

Reply:
done